internal/syscallcompat/emulate_test.go

```diff
@@ -95,6 +104,15 @@ func TestEmulateRenameat(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Test with absolute path
+	err = emulateRenameat(-1, tmpDir + "/dir2/f2", -1, tmpDir + "/dir2/f1")
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = os.Stat(tmpDir + "/dir2/f1")
```

Comment:
Oh, small typo here.

Reply:
Or not ... sorry for the noise (thought the path was incorrect, but should be fine - otherwise the tests also wouldn't pass)